### DISCUSSION TITLE
How to search for string `ABCDE__`?

### DISCUSSION THREAD

**Discussion by molleweide** [BurntSushi/ripgrep#2098] [Q&A] [answered]
Hi there guys,

I am a beginner with ripgrep and I don't understand how to search for a string with a special char such as these:
`_ABCDE`, `ABCDE**`, or `_ABCDE*F`

I have done some searching and reading under the link to rust regex but it is not obvious to me how I do this?

**Comment by BurntSushi** [comment, marked as answer]
You can either use the `-F` flag or escape meta characters. For example, `AB\*CD`.

**Reply by molleweide** [reply]
Thanks for replying!

I'll test this and read some more.
